playground/ProxylessEndToEnd/ProxylessEndToEnd.AppHost/aspire-manifest.json

```diff
@@ -9,6 +9,7 @@
           "scheme": "tcp",
           "protocol": "tcp",
           "transport": "tcp",
+          "port": 9999,
```

Comment:
I assume/know this isn't caused by your change, but are these ports supposed to go into the manifest?

Reply:
Yes. I just happen to be the one that regenerates all the manifests ;)